### DISCUSSION TITLE
Welcome to Synology_enable_eunit Discussions!

### DISCUSSION THREAD

**Discussion by 007revad** [007revad/Synology_enable_eunit#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Tell us if the script worked for you and which model NAS and expansion unit you have.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LMQTLV** [comment]
Hello,

Can you tell me if it is possible to schedule it, as for "Synology HDD db"?
I would like to try something like that, which works perfectly on the command line, but I don't dare schedule it :
**`echo -ne '4\n' | ./syno_enable_eunit.sh --autoupdate=3`**

Nota : "4" is the option for a DX213.

Thank you :)

**Reply by 007revad** [reply]
I actually though of adding this months ago but must have forgotten about it.

I'll do it today. It will probably be like:
`./syno_enable_eunit.sh --autoupdate=3 --unit=dx213`

**Reply by 007revad** [reply]
Done 😄 

See the readme https://github.com/007revad/Synology_enable_eunit for usage info.

Download v2.1.11 https://github.com/007revad/Synology_enable_eunit/releases

**Reply by Caszie** [reply]
I finally got my DX513 working on my 1522.
(syno_enable_eunit.sh -e --unit=EUNIT)

Just one "dumb question": What does the "autoupdate=3" do?

**Reply by LMQTLV** [reply]
> I finally got my DX513 working on my 1522. (syno_enable_eunit.sh -e --unit=EUNIT)
> 
> Just one "dumb question": What does the "autoupdate=3" do?

Automatic update every 3 days => [https://github.com/007revad/Synology_enable_eunit](https://github.com/007revad/Synology_enable_eunit)  ;-) 

```
  -c, --check           Check expansion units status
  -r, --restore         Restore from backups to undo changes
      --unit=EUNIT      Automatically enable specified expansion unit
                          Only needed when script is scheduled
                          EUNIT is dx517, dx513, dx213, dx510, rx418, rx415,
                          rx410, rx1217rp, rx1217, rx1214r, rx1214, rxX1211rp,
                          rx1211, dx1215ii, dx1215 or dx1211
  -e, --email           Disable colored text in output scheduler emails
      --autoupdate=AGE  Auto update script (useful when script is scheduled)
                          AGE is how many days old a release must be before
                          auto-updating. AGE must be a number: 0 or greater
  -h, --help            Show this help message
  -v, --version         Show the script version
```

**Reply by Caszie** [reply]
Thnx

**Comment by LMQTLV** [comment]
Hello,
Thank you very much, you're a chief 🥇 
😄 

I just bought you a coffee to thank you for your work.

**Comment by Crashoveride9579** [comment]
Hello,

I don't succed to make it works on my 923+.
Here my configuration : 
![image](https://github.com/user-attachments/assets/dd2b34a6-efb3-4c58-a8c3-27cfaa35683d)
![image](https://github.com/user-attachments/assets/4eb7c6d9-4d6d-4a25-873e-8b066ad22098)
![image](https://github.com/user-attachments/assets/eee573e6-8222-4263-895e-08415cc44ab6)
![image](https://github.com/user-attachments/assets/3f1c39e3-c401-4b4c-94fa-223e21a754d9)
![image](https://github.com/user-attachments/assets/72e188dd-06ca-4fc5-bec4-2b3700da5031)


I certainly maked something wrong, but what ? ^^'

Thank's for help ! :)

**Comment by LMQTLV** [comment]
Hello,
You don't have to use the `cd `instruction or double quotes.
Write something like that instead :
`/volume2/HugoPerso/........./syno_enable_eunit.sh -e --unit=dx513`

**Reply by Crashoveride9579** [reply]
Oh, thank you !
